data/recommendation/style.scss

```diff
@@ -34,18 +34,16 @@ body {
     width: 100%;
   }
   .image-box {
-    width: 10%;
+    align-items: center;
+    display: flex;
```

Comment:
I think what you had here before is fine, `margin: auto` on the image. You could also set `text-align: center` on the parent. What was the problem before this PR?